blog/content/post/2021-10-03-fobnail-promotion.md

```diff
@@ -0,0 +1,266 @@
+companies like Microsoft are pushing the silicon, hardware and firmware vendors
+to enhance the platform security by establishing [Secure Core PC](https://www.microsoft.com/en-us/windowsforbusiness/windows10-secured-core-computers)
+standard and for example requiring the TPM in Windows 11. For example one of
+the requirements of the Secure Core PC is the use of Dynamic Root of Trust for
+Measurement technology in order to measure the operating system software in the
+processor trusted execution environment impenetrable from external attack
+vectors. With such measurements one may use remote attestation to ensure the
+software that is running on the machine has not been tampered with, without
+relying on the possibly buggy firmware. What is more important Secured Core PCs
+aim to provide such high level security out-of-the-box as much as possible.
+This is very important because the security is still hard to achieve in an
+easy and straightforward way. To eliminate such hardships and obstacles open
+source projects emerged to simplify whole process of boot process hardening:
+
+- [safeboot](https://safeboot.dev/) - leverages firmware Static Root of Trust
+  for Measurement to secure the boot process of Linux OS with UEFI Secure Boot
+- [heads](https://osresearch.net/) - leverages firmware Static Root of Trust
+  for Measurement and Linux kernel as a part of open source boot firmware to
+  perform verification of the software components of the operating system with
+  the helps of TPM and USB security tokens. Thanks to open source boot firmware
+  the risk of buggy firmware implementation is significantly reduced.
+
+But do those projects and solve all the problems? What to do with those
+measurements? The short answer is "remote attestation". There is a need of a
+trusted entity that will tell the machine owner that the measured software has
+the approved cryptographic footprints (measurements). At 3mdeb we aim to create
+an open hardware USB security token called [Fobnail](https://fobnail.3mdeb.com/)
+which will act as a axiomatically trusted device and provide attestation
+services to avoid the use of network and potentially untrusted attestation
+servers or man-in-the-middle network attacks. The project became possible thanks
+to the sponsorship of the [NLnet Foundation](https://nlnet.nl/project/Fobnail/).
+
+## safeboot
+
+Before we start explaining how Fobnail may improve the security of the boot
+process lets' review and compare the open source boot process hardening
+projects starting with.
+
+First of all safeboot is a set of scripts and wrappers that help automate and
+speed up provisioning of the machine to leverage TPM and UEFI Secure Boot full
+potential. In short it packs Linux kernel, initial ramdisk (with safeboot
+scripts) and commandline parameters into a single EFI file and creates two
+copies of it for normal boot flow and recovery. Additionally the root
+filesystem is being encrypted and the password is being sealed to TPM and
```

Comment:
_"filesystem is encryprted with the password sealed to the TPM"_

Reply:
Fixed